packages/languages/src/browser/language-client-contribution.ts

```diff
@@ -80,6 +82,10 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
 
     protected createLanguageClient(): ILanguageClient {
         const clientOptions = this.createOptions();
+        clientOptions.initializationFailedHandler = () => {
```

Comment:
How often does it happen? For example, if the connection is lost `LanguageClient` tries to reconnect 5 times will a user see 5 notifications?

Reply:
It happens 2-5 times from what I saw. Before this patch, it displays "Connection got
disposed" 2-5 times. With the patch, it displays "Failed to start language server $NAME" which is at least clearer.